Kernel/Arch/riscv64/CSR.h

```diff
@@ -356,6 +383,8 @@ struct AK::Formatter<Kernel::RISCV64::CSR::SCAUSE> : AK::Formatter<FormatString>
             return builder.put_string("Supervisor timer interrupt"sv);
         case Kernel::RISCV64::CSR::SCAUSE::SupervisorExternalInterrupt:
             return builder.put_string("Supervisor external interrupt"sv);
+        case Kernel::RISCV64::CSR::SCAUSE::CounterOverflowInterrupt:
+            return builder.put_string("Counter overflow interrupt"sv);
```

Comment:
Please use the exact spelling from the RISC-V ISA manual, so "Counter-overflow interrupt".